src/components/CustomNavLink.js

```diff
@@ -0,0 +1,11 @@
+import { NavLink } from "react-router-dom";
+
+function CustomNavLink({label, path}) {
+    return (
+        <NavLink className={({isActive}) => "font-bold uppercase " + (isActive ? "text-primary-200" : "")} to={path}>
```

Comment:
Let's write `className` and `to` on separate lines to make it more readable

Reply:
forgot to use prettier, I'll fix it